index.rst

```diff
@@ -0,0 +1,190 @@
+..  Content of technical report.
+
+  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
+  for a guide to reStructuredText writing.
+
+  Do not put the title, authors or other metadata in this document;
+  those are automatically added.
+
+  Use the following syntax for sections:
+
+  Sections
+  ========
+
+  and
+
+  Subsections
+  -----------
+
+  and
+
+  Subsubsections
+  ^^^^^^^^^^^^^^
+
+  To add images, add the image file (png, svg or jpeg preferred) to the
+  _static/ directory. The reST syntax for adding the image is
+
+  .. figure:: /_static/filename.ext
+     :name: fig-label
+     :target: http://target.link/url
+
+     Caption text.
+
+   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
+   See the README at https://github.com/lsst-sqre/lsst-report-bootstrap or
+   this repo's README for more info.
+
+   Feel free to delete this instructional comment.
+
+:tocdepth: 1
+
+This document summarizes the work that was done to test the accuracy of shear measurement using the measurement algorithms in the DM package meas_modelfit.git.  This package contains the Python package lsst.meas.modelfit, which houses both the CModel algorithm -- used to measure galaxy shapes, and the ShapeletPsfApprox Algorithm (SPA) -- used to create a parameterized model of the point spread function (Psf).
+
+The images used for this test were created by GalSim simulations as described below. The galaxy profiles areies parameterized GalSim models. However, the Psfs are not from GalSim models, as we want the framework to accommodate a variety of different realistic and simulated Psfs. A separate psf image for each galaxy is used to simulate the effects of seeing:  this includes both atmospheric and optical effects. 
+
+In these tests, the Psf is modelled using SPA for later use during any measurement process where an estimation of the Psf is required (for example, for Psf weighting).
+
+Our goal is to measure enough simulated galaxies to determine the effect of different Psf approximation and shape measurement algorithms on galaxy shear estimation.
+
+DM-5447a – Techniques used in this simulations
+=================================================
+
+Creating the Psf Library:
+-------------------------
+We used PhoSim to create Psf images for an entire LSST focal plane.  The LSST focal plane is divided into 21 rafts with 9 sensors per raft.  To create a library of Psf images, 10000 positions were selected at random and PhoSim was used to create a stellar image at each position.  After removing unusable stars (those which did not fall fully on any particular sensor, or which were close to another star), between 7000 and 8000 usable stellar images remained for each simulated focal plane.
+
+Psf libraries were created for “raw seeing” values of 0.5, 0.7, and 0.9 arcseconds and for LSST filters (f2 and f3).  Most of the tests during this cycle were run using the f2_0.7 library.
```

Comment:
What are f2 & f3? I'm guessing they correspond to the `filter_2.txt` and `filter_3.txt` files at https://bitbucket.org/phosim/phosim_release/src/HEAD/data/lsst/, but readers shouldn't have to understand the guts of PhoSim to follow your text.

Since PhoSim and the associated filter definitions, etc, are versioned, it would be nice to record which versions you actually used.

Are the libraries created available?